gsoc2018-bharat/src/package/extract.py

```diff
@@ -0,0 +1,3238 @@
+#!/usr/bin/env python
```

Comment:
Do you need this file ? If you are just using this as a preprocessing tool, it might be better to specify this https://github.com/attardi/wikiextractor as a dependency

Reply:
Cool. Will add the link to the README and remove the extract.py script from the repo.